jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -73,12 +97,29 @@ pipeline {
             agent { label agent_nodes["x64"] }
             steps {
                 script {
-                    if (VERSION == '') {
+                    def url_paths = [:]
+
+                    OPENSEARCH_ARTIFACT_URL ? url_paths["opensearch"] = OPENSEARCH_ARTIFACT_URL : null
+                    OPENSEARCH_DASHBOARDS_ARTIFACT_URL ? url_paths['opensearch-dashboards'] = OPENSEARCH_DASHBOARDS_ARTIFACT_URL : null
+
+                    if (OPENSEARCH_DASHBOARDS_ARTIFACT_URL && OPENSEARCH_ARTIFACT_URL == "") {
                         currentBuild.result = 'ABORTED'
-                        error('VERSION cannot be empty. Please provide version of the artifacts to validate')
+                        error("Provide OPENSEARCH_ARTIFACT_URL to validate")
+                    }
+
+                    if(OPENSEARCH_ARTIFACT_URL) {
+                        ARCHITECTURE = url_paths.values().find { it.contains("x64") } ? "x64" : "arm64"
+                        DISTRIBUTION = url_paths.values().find { it.contains("tar.gz") } ? "tar" : (url_paths.values().find { it.contains("rpm") } ? "rpm" : "yum")
```

Comment:
Did we miss debian here?

Reply:
The validation workflow currently supports only tar, rpm, yum and docker. So I didn't include windows and debian.